tinker-actions/src/kernelupgrd/kernel_upgrade.sh

```diff
@@ -50,19 +50,19 @@ mount --bind /etc/resolv.conf /mnt/etc/resolv.conf
 
 mv /mnt/etc/apt/apt.conf.d/99needrestart /mnt/etc/apt/apt.conf.d/99needrestart.bkp 
 
+#Enter into Ubuntu OS for the latest 6.x kernel instalation
+chroot /mnt /bin/bash <<EOT
+
+apt update
+
 #Get the Latest canonical 6.8 kerner version 
-export kernel_version=$(chroot /mnt /bin/bash -c "apt-cache search linux-image | grep 'linux-image-6.8.*-generic' | tail -1 | awk '{print \$1}' | grep -oP '(?<=linux-image-)[0-9]+\.[0-9]+\.[0-9]+-[0-9]+'")
+export kernel_version=$(/bin/bash -c "apt-cache search linux-image | grep 'linux-image-6.8.*-generic' | tail -1 | awk '{print \$1}' | grep -oP '(?<=linux-image-)[0-9]+\.[0-9]+\.[0-9]+-[0-9]+'")
```

Comment:
Please fix up kernel version to `6.8.0-55-generic` as CO team confirmed that it is working with patch version 55.